C/Bit--Manipulation/Swap_even_and_odd_bits.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+
+unsigned int swapBits(unsigned int x)
+{
+    unsigned int evenBits = x & 0xAAAAAAAA;
+    /* 0x is written here. It means it is hexadecimal number and computer stores hexadecimal number in binary format. 
+    Also, A is written here 8 times. So, it means it will store 32 bits integer i.e., (8*4=32) . Each A will be replaced by 1010 . 
+    So, in that way 32 bit integer  will be stored. 
+    */
+
+    /*A in binary is 1010 . So, by performing bitwise & operation, we will get all the even bits of number x since, 1 is 
+   present at even positions. (Masking) */
+
+    unsigned int oddBits = x & 0x55555555; //5 in decimal is 0101. So, by performing bitwise & operation, we will get
+                                           // all the odd bits of number x since 1 is present at odd positions.
+                                           //(Masking)
+
+    evenBits >>= 1; // Right shifting even bits
```

Comment:
Shift the comment above the line of code.

Reply:
Yes, I have done it. Now, you can review my code again.

Reply:
No. You have not resolved it yet